plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -67,9 +67,11 @@
 _ABI = "abi"
 _BYTECODE = "bytecode"
 EIP1559 = "eip1559"
+EIP1559_POLYGON = "eip1559_polygon"
 GAS_STATION = "gas_station"
-AVAILABLE_STRATEGIES = (EIP1559, GAS_STATION)
-
+AVAILABLE_STRATEGIES = (EIP1559, GAS_STATION, EIP1559_POLYGON)
+SPEED_FAST = "fast"
```

Comment:
Are we gonna need any other strategies or `fast` will do it?

Reply:
That is the safest one to ensure that the txs will go through, and since polygon is very cheap I'd stuck to it.